trials.py

```diff
@@ -0,0 +1,9 @@
+from sympy import *
```

Comment:
This file should not be here. Instead add a `test_order_reducing_substitution` function to `sympy/solvers/tests/test_ode.py`

Reply:
Right it got added.I need to learn that software or git from command line.